Build errors

In the Readme.md, section **Cracking unknown ring and rotor positions**, there's a code
```
EnigmaBase base {ETW_ABCDEF, m3_IV, m3_V, m3_II, UKWB};

HighScores scores = smart_decipher(base, plug, ct);
```

But `UKWB` is undefined. It seems `m3_UKVB` should be used instead. And when compiling, I get the following errors:
```
.../include/bruteforce-enigma.h:352:66: error: no matching function for call to 'smart_decipher_positions<std::vector<int, std::allocator<int> >, 10, double (*)(const std::vector<int, std::allocator<int> >&)>(const basic_enigma_base<EnigmaKey>&, const Plugboard&, const std::vector<int>&, double (*&)(const std::vector<int>&))'
...
.../include/bruteforce-enigma.h:354:7: error: could not convert '{ct, <expression error>}' from '<brace-enclosed initializer list>' to 'HighScores<10, EnigmaKey>'

```
I'm compiling under Windows with TDM-GCC-64 (g++ v10.3.0). Sorry, I'm almost total zero in C++. What should I do to fix errors?

--- a/README.md
+++ b/README.md
@@ -39,7 +39,7 @@ The smart decipher should be sufficient for almost all ciphers. It uses a simila
 Ciphertext ct = "NPNKANVHWKPXORCDDTRJRXSJFLCIUAIIBUNQIUQFTHLOZOIMENDNGPCB";
 Plugboard plug = "";
 
-EnigmaBase base {ETW_ABCDEF, m3_IV, m3_V, m3_II, UKWB};
+EnigmaBase base {ETW_ABCDEF, m3_IV, m3_V, m3_II, m3_UKWB};
 
 HighScores scores = smart_decipher(base, plug, ct);
 scores.print();

--- a/include/bruteforce-enigma.h
+++ b/include/bruteforce-enigma.h
@@ -349,7 +349,7 @@ HighScores<N, KeyType> smart_decipher (
 )
 {
      std::puts("Breaking enigma...");
-     return {ct, smart_decipher_positions<std::vector<int>, N, F>(
+     return {ct, smart_decipher_positions<KeyType, std::vector<int>, N, F>(
           base, plugboard, ct.ordinals, scoring_function
      )};
 }

